butterknife-compiler/src/main/java/butterknife/compiler/ButterKnifeProcessor.java

```diff
@@ -201,6 +217,10 @@
     return false;
   }
 
+  public HashMap<String, List<Element>> getMapGeneratedFileToOriginatingElements() {
```

Comment:
This method is also always visible for testing. Of course, we can change the visibility accordingly.